topostats/plottingfuncs.py

```diff
@@ -646,6 +662,45 @@ def add_bounding_boxes_to_plot(fig, ax, shape: tuple, region_properties: list, p
     return fig, ax
 
 
+def number_grain_plots(
+    fig, ax, shape: tuple, region_properties: list, pixel_to_nm_scaling: float, offset: tuple
+) -> tuple:
+    """
+    Add the grain numbers to the plot.
+
+    Parameters
+    ----------
+    fig : plt.figure.Figure
+        Matplotlib.pyplot figure object.
```

Comment:
These could have markup so they are rendered differently.

```suggestion
        ``Matplotlib.pyplot`` figure object of labelled grains to be numbered.
```

Same applies for other references.